### ISSUE TITLE
BUG: `sysconfig._init_posix` fails on aarch/ppc

### ISSUE THREAD

**Issue by h-vetinari** [conda-forge/python-feedstock#536]
Not sure if this is an issue with setuptools or python itself, but the last 2 days I've seen (for example in [this](https://github.com/conda-forge/numpy-feedstock/pull/237) PR; [CI run](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=431370&view=results)):
```
  Traceback (most recent call last):
    File "/home/conda/feedstock_root/build_artifacts/numpy_1640213318070/_build_env/venv/lib/python3.7/site-packages/setuptools/_distutils/sysconfig.py", line 489, in _init_posix
      _temp = __import__(name, globals(), locals(), ['build_time_vars'], 0)
  ModuleNotFoundError: No module named '_sysconfigdata_m_linux_powerpc64le-linux-gnu'
```

The reason I'm opening it here nevertheless is because pypy on aarch/ppc doesn't run into this issue.

**Comment by isuruf** [comment]
This was broken with new setuptools and fixed with crossenv=1.1.4

**Comment by h-vetinari** [comment]
Great, thanks a lot! :)
